build/win32/code.iss

```diff
@@ -68,7 +68,7 @@ Name: "{commondesktop}\{#NameLong}"; Filename: "{app}\{#ExeBasename}.exe"; AppUs
 Name: "{userappdata}\Microsoft\Internet Explorer\Quick Launch\{#NameLong}"; Filename: "{app}\{#ExeBasename}.exe"; Tasks: quicklaunchicon; AppUserModelID: "{#AppUserId}"
 
 [Run]
-Filename: "{app}\{#ExeBasename}.exe"; Description: "{cm:LaunchProgram,{#NameLong}}"; Tasks: runcode; Flags: nowait postinstall; Check: WizardSilent
+Filename: "{app}\{#ExeBasename}.exe"; Description: "{cm:LaunchProgram,{#NameLong}}"; Tasks: runcode; Flags: nowait postinstall skipifsilent; Check: WizardSilent
```

Comment:
Based on `Check: WizardSilent` i think this line only runs when the installation is silent anyway. Looking at the [VS Code version of this file](https://github.com/Microsoft/vscode/blob/master/build/win32/code.iss) it seems like they just removed this line.

Clearly this works anyway so it's fine with me, but it looks like we might be able to just remove the whole line